src/components/UserIndicator.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+
+const propTypes = {
+    isOffline: PropTypes.func.isRequired,
+};
+
+const UserIndicator = ({isOffline}) => {
+    const onlineStyle = isOffline ? styles.statusIndicatorOffline : styles.statusIndicatorOnline;
+    return (
+        <div
```

Comment:
This works for this one case, but the solution to replace the `View` with a `div` is not acceptable. As per the original issue:

>This should be solved holistically, not as a one-off solution for each of the problems. The graphical elements are supposed to be absolute positioned, but on initial page load, this is being overruled by a position:relative

Changing this from a `View` to a `div` means that anytime this situation occurs we will have to create a special version of that component that uses a `div` instead of a `View` so the styles are respected correctly.

Reply:
Hi, marcaaron
Thanks for your feedback.
As I write in description, in that case, react-native remove alignItems and justifyContent styles. Because of this reason, some icons has misalignment. But can we change react native library itself? I think this is impossible and we should use native component such as ```div``` if needed.